JASP-Engine/JASP/R/correlationbayesian.R

```diff
@@ -766,11 +768,12 @@ CorrelationBayesian <- function(jaspResults, dataset=NULL, options, ...) {
     return(jaspResults[[indexName]]$object)
 
   results <- list()
+  .setSeedJASP(options)
   for (method in names(bfObject))
     results[[method]] <- bstats::computeCorPosteriorLine(bfObject = bfObject[[method]], method = method, alternative = options[["alternative"]])
```

Comment:
If you do this in the for loop then the outcome of a test will be the same regardless of the number of tests

Reply:
Thanks, I missed that before.